test/fixtures/plugins/train-test-fixture/README.md

```diff
@@ -0,0 +1,5 @@
+This is a very simple train transport plugin. It provided fixed responses to file.content and command.stdout/stderr/exit_status.
```

Comment:
`Train`?

Reply:
This is an internal test fixture. Its value lies in its very simple functionality.  I feel it is not a good use of resources to review and correct for comments.